FWCore/MessageLogger/interface/MessageSender.h

```diff
@@ -29,8 +18,10 @@ namespace edm {
 
   public:
     // ---  birth/death:
-    MessageSender() : errorobj_p() {}
+    MessageSender() = default;
     MessageSender(ELseverityLevel const& sev, std::string_view id, bool verbatim = false, bool suppressed = false);
+    MessageSender(MessageSender&&) = default;
+    MessageSender(MessageSender const&) = default;
```

Comment:
How about defaulting or deleting the copy and move assignment operators explicitly?

Reply:
done